templates/kubernetes/docs/backups.md

```diff
@@ -1,69 +1,59 @@
 ---
-wrapper_template: base_docs.html
-context:
-    title: Backups
+wrapper_template: "base_docs.html"
 markdown_includes:
-    nav: shared/_side-navigation.md
+  nav: "shared/_side-navigation.md"
+context:
+  title: "Backups"
+  description: How to backup and restore the state of your Kubernetes cluster in the etcd datastore.
 ---
 
-The state of your **Kubernetes** cluster is kept in the **etcd** datastore. This page
-shows how to backup and restore the etcd included in the **Canonical
-Distribution of Kubernetes <sup>&reg;</sup>**.
+The state of your **Kubernetes** cluster is kept in the **etcd** datastore. This page shows how to backup and restore the etcd included in the **Canonical Distribution of Kubernetes <sup>&reg;</sup>**.
 
- Backing up application specific data, normally stored in a persistent volume, is not
- currently supported by native Kubernetes. Various third party solutions are available -
- please refer to their own documentation for details.
+Backing up application specific data, normally stored in a persistent volume, is not currently supported by native Kubernetes. Various third party solutions are available -
+please refer to their own documentation for details.
 
- ## Creating an **etcd** snapshot
+## Creating an **etcd** snapshot
 
- <div class="p-notification--warning"><p class="p-notification__response">
+ <div class="p-notification--warning"><p markdown="1" class="p-notification__response">
```

Comment:
What does `markdown="1"` do?

Reply:
it tells the parser to parse the text inside the block

Reply:
not sure it works